app/policies/idv/gpo_verify_by_mail_policy.rb

```diff
@@ -42,6 +43,12 @@ def disabled_for_biometric_comparison?
       resolved_authn_context_result.two_pieces_of_fair_evidence?
     end
 
+    def disabled_for_ipp?
+      return false unless IdentityConfig.store.no_verify_by_mail_for_biometric_comparison_enabled
+
+      user.has_in_person_enrollment?
+    end
+
```

Comment:
@matthinz Is this just an additional check to ensure GPO is not offered for IPP? I feel like I am not able to hit this code because the the Phone view is conditionally displaying the verify by mail option already.